dspace/config/dspace.cfg

```diff
@@ -2478,6 +2478,13 @@ solr.default.filter.override=location
 # This configuration is set by the Maven profiles:
 submit.journal.config=${default.submit.journal.config}
 
+submit.journal.credential.dir = ${dspace.dir}/submission/credential
+
+# Copy the json line for this property from dev.datadryad.org/gmail.settings.xml to your maven settings.xml
+submit.journal.clientsecrets = ${default.submit.journal.clientsecret}
```

Comment:
Can you explain what this is and how to generate it? Presumably it's linked to someone's google account and could be revoked.